source/constructs/lib/back-end/back-end-construct.ts

```diff
@@ -109,6 +106,22 @@ export class BackEnd extends Construct {
         DEFAULT_FALLBACK_IMAGE_BUCKET: props.fallbackImageS3Bucket,
         DEFAULT_FALLBACK_IMAGE_KEY: props.fallbackImageS3KeyBucket,
       },
+      depsLockFilePath: path.join(__dirname, "../../../image-handler/package-lock.json"),
+      bundling: {
+        externalModules: ["sharp"],
```

Comment:
I'd imagine that `aws-sdk` should really be a `dependency` of this module, not a `devDependency`. You can add it as an external module and move it to the correct place and it still won't get bundled.

Reply:
addressed